Library/Formula/collada-dom.rb

```diff
@@ -3,33 +3,51 @@
 class ColladaDom < Formula
   desc "C++ library for loading and saving COLLADA data"
   homepage 'http://www.collada.org/mediawiki/index.php/Portal:COLLADA_DOM'
-  url 'https://downloads.sourceforge.net/project/collada-dom/Collada%20DOM/Collada%20DOM%202.4/collada-dom-2.4.0.tgz'
-  sha1 '74e28d670497abc897c06a41df7d28eea2bac836'
+  url 'https://github.com/rdiankov/collada-dom/archive/v2.4.3.tar.gz'
+  sha1 '0eb3b9f3183de8e5183f509311aaa7e8bc2679b7'
 
   depends_on 'cmake' => :build
   depends_on 'pcre'
   depends_on 'boost'
 
-  # Fix build of minizip: quoting arguments to cmake's add_definitions doesn't work the way they thought it did.
-  patch :DATA
-
   def install
     system "cmake", ".", *std_cmake_args
     system "make install"
   end
-end
 
-__END__
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 72b6deb..0c7f7ce 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -100,7 +100,7 @@ endif()
+  def test
+    # Test per: https://www.khronos.org/collada/wiki/DOM_guide:_Creating_documents#Creating_a_simple_document
```

Comment:
We can probably get by without the comment (unless attribution is absolutely required!)